src/App.tsx

```diff
@@ -14,11 +15,82 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  good: string,
+  selectedGood: string,
+};
+
+class App extends React.Component<{}, State> {
+  state: State = {
+    good: `${goodsFromServer.find(x => x === 'Jam')}`,
```

Comment:
there will be 'Jam' from goodsFromServer. If you know that you want Jam, why did you not use the 'Jam' string without find?

```suggestion
    good: 'Jam',
```